common/middleware/collection/set-pagination.js

```diff
@@ -1,5 +1,6 @@
 const { format } = require('date-fns')
 
+const DATESELECT = require('../../../app/date-select/constants')
```

Comment:
Is this the first instance of `common` reaching into the app directory?

Reply:
Unfortunately not

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/helpers/field/index.js#L11-L12

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/middleware/locals.js#L3

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/3516af7a7acdf8020b9d4327b71de88edfd794a9/common/presenters/move-to-meta-list-component.js#L3